src/Utility.js

```diff
@@ -93,8 +93,17 @@ export function sizeof (value) {
 /**
  * @param {any[]} value
  * @param {any} element
- * @return {number}
+ * @return {void}
```

Comment:
return value has stayed unused so I thought it might be better to skip returning a value at all to save few bytes from the script